test/OpenTelemetry.Tests/Metrics/AggregatorTest.cs

```diff
@@ -188,6 +188,49 @@ public void HistogramWithOnlySumCount()
             Assert.False(enumerator.MoveNext());
         }
 
+        [Fact]
+        public void MultiThreadedHistogramUpdateAndSnapShotTest()
```

Comment:
see if this can modified : https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/test/OpenTelemetry.Tests/Metrics/MetricAPITest.cs#L1401

Reply:
Thanks for sharing. I took the above mentioning test as an example.
Having the test in AggregatorTest serves more as a unit test; and having it at MetricAPITest sounds more of a integration test.
Either works for me!